src/ansys_sphinx_theme/extension/autoapi.py

```diff
@@ -0,0 +1,110 @@
+"""Module for adding the autoapi extension with the theme."""
+
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import os
+from typing import Any, Dict
+
+from sphinx.application import Sphinx
+
+from ansys_sphinx_theme import __version__, get_autoapi_templates_dir_relative_path
+
+
+def add_autoapi_theme_option(app: Sphinx) -> None:
+    """Add the autoapi template path to the theme options.
+
+    Parameters
+    ----------
+    app : ~sphinx.application.Sphinx
```

Comment:
Is "sphinx" defined in the intersphinx dictionary for the doc build of this project? Otherwise the rendered docs won't be able to point to it.

Reply:
yeah sure, i will check and add.